spec/lib/player_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+require 'spec_helper'
+
+describe Scrabble::Player do
+
+  describe "class methods" do
+    it "responds to 'new'" do
+      expect(Scrabble::Player).to respond_to :new
+    end
+
+
+  end
+
+  describe "attributes" do
+    let(:player) { Scrabble::Player.new("Voltron") }
+
+    it "has a name" do
+      expect(player.name).to eq("Voltron")
+    end
+
+    it "has a 'plays' attribute" do
+      expect(player).to respond_to :plays
+    end
+  end
+
+  describe "instance methods" do
+    let(:player) { Scrabble::Player.new("Voltron") }
+
+    it "responds to 'play'" do
+      expect(player).to respond_to :play
+    end
+
+    it "responds to 'total_score'" do
+      expect(player).to respond_to :total_score
+    end
+
+    # # I didn't get total_score working. This spec doesn't pass, so I commented it out for now. 
+    # let(:plays) { Scrabble::Player.plays(["hello", "medic", "qat"]) }
```

Comment:
Be careful of the way you set up your test. `.plays` is an instance variable on `Scrabble::Player`, so you'll get an error if you call it directly on the class. Also, since it is an instance variable, it doesn't take any arguments.

I believe you want to simulate the situation where an instance of `Scrabble::Player` actually plays three words. What method should you call to set that up (and thus populate the `@plays` array?)